### ISSUE TITLE
[SF][AnalyticalTable]: A11y - Table header text should not be empty

### ISSUE THREAD

**Issue by summer2200** [SAP/ui5-webcomponents-react#6209]
### Describe the bug

We've received another A11y issue found by axe tool, the "Select All" checkbox is a table header but it does not have any visible text.
The solution could be either add aria-hidden="true" to the table header cell of checkbox, or add some invisible text to it so screen reader can detect.
<img width="1272" alt="Screenshot 2024-08-15 at 16 55 26" src="https://github.com/user-attachments/assets/521a7e9a-91b5-439b-8ad0-c5292894b998">

### Reproduction steps

1.
2.
3.
...

### UI5 Web Components for React Version

1.29.8

### UI5 Web Components Version

1.24.8

### Browser

Chrome

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by summer2200** [comment]
Hi @Lukas742 ,

Thanks for fixing this issue! Now the check box has aria-hidden="true" and it's expected. But actually aria-hidden="true" needs to be added to the table header cell element of the checkbox too, because the cell element is a table header and it does not have any text inside and it's being detected by AXE tool. 
Please see below screenshot:
![Screenshot 2024-10-21 at 14 17 20](https://github.com/user-attachments/assets/19b04cfb-a7bc-4632-8ce9-18683192367c)


Do you want me to raise another issue for this?

**Comment by summer2200** [comment]
Hi @Lukas742 ,

Thanks for fixing this issue! Now the check box has aria-hidden="true" and it's expected. But actually aria-hidden="true" needs to be added to the table header cell element of the checkbox too, because the cell element is a table header and it does not have any text inside and it's being detected by AXE tool. 
I have raised another issue to track it, please help check, thanks!
